### PR TITLE
Feature/auto complete part 1

### PR DESCRIPTION
**Pull request by code2nguyen**
- [x] Fix Conflicts
- [x] Story in ts
- [x] Add placeholder
- [x] Fix Build
- [x] Fix placeholder text color

### CODE REVIEW

**Review by dorsaffrigui — COMMENTED** [review]
You can use the text-color mixin to give a color to your item name. 

Screenshot 2022-02-01 at 16 54 15


Can we  add  a  default placeholder to your text field ?

File: libs/web-components/autocomplete/stories/autocomplete.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  title="Components/Autocomplete"
+  component={'fds-autocomplete'}
+  argTypes={{
+    icon: {
+      table: {
+        category: 'properties',
+        defaultValue: { summary: '' }
+      },
+      control: 'text',
+      defaultValue: '',
```
Comment (bcldvd) [inline libs/web-components/autocomplete/stories/autocomplete.stories.mdx:18]: `defaultValue` will be deprecated in Storybook 7. Can we remove it and rely only on args and summary ?

File: libs/web-components/autocomplete/src/autocomplete.ts [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  @property({ type: String }) icon = '';
+  @property({ type: Boolean }) loading = false;
+  @property({ type: Boolean }) showClearButton = false;
+  @property({ type: String }) value = '';
+  @property({ type: Number }) minLengthToOpenMenu = 0;
+  @property({ type: Boolean }) required = false;
+  @property({ type: Boolean }) useInnerFilter = true;
+  @property({ type: Boolean }) disabled = false;
+
+  /** @soyPrefixAttribute */
```
Comment (bcldvd) [inline libs/web-components/autocomplete/src/autocomplete.ts:38]: Remove  `/** @soyPrefixAttribute */`

**Comment by code2nguyen** [comment]
> 

- Fixed color
- Placeholder is specific for each context. We don't need or have a generic placeholder.

File: libs/web-components/autocomplete/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Autocomplete
```
Comment (dorsaffrigui) [inline libs/web-components/autocomplete/README.md:1]: Could you please document the README ?

**Review by bcldvd — APPROVED** [review]

**Review by dorsaffrigui — APPROVED** [review]

**Pull request merged**